ocaml/idl/templates/relationships.mustache

```diff
@@ -0,0 +1,19 @@
+---
+ layout: doc
+---
+
+# Relationships Between Classes
+
+Fields that are bound together are shown in the following table:
+
+|_object.field_                          |_object.field_                          |_relationship_ |
+|:---------------------------------------|:---------------------------------------|:--------------|
+{{#relationships}}
+|{{{a_field}}}|{{{b_field}}}|{{relationship}}|
+{{/relationships}}
+
+The following figure represents bound fields (as specified above)
+diagramatically, using crow's foot notation to specify one-to-one,
+one-to-many, or many-to-many relationships:
+
+![Class relationships](classes.png 'Class relationships')
```

Comment:
I'm a bit confused, where is this image in the source? I can see it on https://xapi-project.github.io/xen-api/ but not here. :/

Reply:
It's autogenerated from the .dot file. The internal pipeline does it, I haven't looked into github actions because that was done a long time ago (and I'll consider it out of scope for this PR).